ci/scripts/run-check_ci.sh

```diff
@@ -23,7 +24,7 @@ pslot=${2:-${pslot:-?}}        # Name of the experiment being tested by this scr
 #         └── ${pslot}
 # Two system build directories created at build time gfs, and gdas
 # TODO: Make this configurable (for now all scripts run from gfs for CI at runtime)
-HOMEgfs="${TEST_DIR}/gfs"
+HOMEgfs="${TEST_DIR}/${SYSTEM_BUILD_DIR:-global-workflow}"
```

Comment:
This default will never be used. L12 already errors if `$SYSTEM_BUILD_DIR` is not set.